README.md

```diff
@@ -460,11 +460,23 @@ So if not already done, install (from this folder):
 
 Then run the server with the test configuration:
 
-`serial-keel py/test-concurrent.ron`
+`serial-keel py/sample/test-concurrent.ron`
 
 #### Python setup
 
-Do a `pip install -r py/requirements.txt` if deps are missing.
+Eventually, once the python package is published to PyPI, you should just be able to install using pip.
```

Comment:
We should remove language that refers to SK not being on pypi, since the assumption is that when this is merged we publish.